qiskit/pulse/builder.py

```diff
@@ -0,0 +1,1927 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""Use the pulse builder to program pulse programs with an imperative
+assembly-like syntax.
+
+To begin pulse programming we must first initialize our program builder
+context with :func:`build`, after which we can begin
+adding program statements. For example, below we write a simple program that
+:func:`play`\s a pulse:
+
+.. jupyter-execute::
```

Comment:
I didn't think module level docstrings were rendered in the docs?

Reply:
I should download the docs and check it out myself:)

Reply:
I think they are now :).